Providing a topology directory fails if there is a sub-directory inside

**Describe the bug**
Julie fails if there we rely on a directory for `--topology` and there is a sub-directory inside, e.g. `schemas`

**To Reproduce**
Steps to reproduce the behavior:
1. Create a topology directory with multiple descriptors inside
2. Add a sub-directory inside, e.g. `schemas`
3. Execute Julie with `--topology MyDirectory`
4. See `Caused by: java.io.FileNotFoundException: descriptors/schemas (Is a directory)`

**Expected behavior**
Sub-directory has either ignored or browse recursively

--- a/src/main/java/com/purbon/kafka/topology/TopologyObjectBuilder.java
+++ b/src/main/java/com/purbon/kafka/topology/TopologyObjectBuilder.java
@@ -55,6 +55,7 @@ public class TopologyObjectBuilder {
     if (isDir) {
       Files.list(Paths.get(fileOrDir))
           .sorted()
+          .filter(p -> !Files.isDirectory(p))
           .map(path -> parser.deserialise(path.toFile()))
           .forEach(subTopology -> topologies.add(subTopology));
     } else {

--- a/src/test/java/com/purbon/kafka/topology/JulieOpsTest.java
+++ b/src/test/java/com/purbon/kafka/topology/JulieOpsTest.java
@@ -212,4 +212,32 @@ public class JulieOpsTest {
     verify(topicManager, times(1)).apply(anyObject(), anyObject());
     verify(accessControlManager, times(1)).apply(anyObject(), anyObject());
   }
+
+  @Test
+  public void builderRunTestAsFromDirectoryWithSchema() throws Exception {
+    String fileOrDirPath = TestUtils.getResourceFilename("/dir_with_subdir");
+
+    Configuration builderConfig = new Configuration(cliOps, props);
+
+    JulieOps builder =
+        JulieOps.build(
+            fileOrDirPath,
+            builderConfig,
+            topologyAdminClient,
+            accessControlProvider,
+            bindingsBuilderProvider);
+
+    builder.setTopicManager(topicManager);
+    builder.setAccessControlManager(accessControlManager);
+
+    doNothing().when(topicManager).apply(anyObject(), anyObject());
+
+    doNothing().when(accessControlManager).apply(anyObject(), anyObject());
+
+    builder.run();
+    builder.close();
+
+    verify(topicManager, times(1)).apply(anyObject(), anyObject());
+    verify(accessControlManager, times(1)).apply(anyObject(), anyObject());
+  }
 }

--- a/src/test/java/com/purbon/kafka/topology/integration/RBACPRoviderRbacIT.java
+++ b/src/test/java/com/purbon/kafka/topology/integration/RBACPRoviderRbacIT.java
@@ -54,7 +54,6 @@ import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.runners.MockitoJUnitRunner;
-import org.testcontainers.shaded.com.google.common.eventbus.AllowConcurrentEvents;
 
 @RunWith(MockitoJUnitRunner.class)
 public class RBACPRoviderRbacIT extends MDSBaseTest {

--- a/src/test/resources/dir_with_subdir/descriptor-with-others.yml
+++ b/src/test/resources/dir_with_subdir/descriptor-with-others.yml
@@ -1,0 +1,55 @@
+---
+context: "contextOrg"
+source: "source"
+foo: "foo"
+bar: "bar"
+zet: "zet"
+projects:
+  - name: "foo"
+    zookeepers: []
+    consumers:
+      - principal: "User:App0"
+      - principal: "User:App1"
+    producers: []
+    streams:
+      - principal: "User:App0"
+        topics:
+          read:
+            - "topicA"
+            - "topicB"
+          write:
+            - "topicC"
+            - "topicD"
+    connectors:
+      - principal: "User:Connect1"
+        topics:
+          read:
+            - "topicA"
+            - "topicB"
+      - principal: "User:Connect2"
+        topics:
+          write:
+            - "topicC"
+            - "topicD"
+    topics:
+      - name: "foo"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+      - dataType: "avro"
+        name: "bar"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+  - name: "bar"
+    zookeepers: []
+    consumers: []
+    producers: []
+    streams: []
+    connectors: []
+    topics:
+      - dataType: "avro"
+        name: "bar"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"

--- a/src/test/resources/dir_with_subdir/descriptor.yaml
+++ b/src/test/resources/dir_with_subdir/descriptor.yaml
@@ -1,0 +1,61 @@
+---
+context: "contextOrg"
+source: "source"
+projects:
+  - name: "foo"
+    zookeepers: []
+    consumers:
+      - principal: "User:App0"
+      - principal: "User:App1"
+    producers: []
+    streams:
+      - principal: "User:App0"
+        topics:
+          read:
+            - "topicA"
+            - "topicB"
+          write:
+            - "topicC"
+            - "topicD"
+    connectors:
+      - principal: "User:Connect1"
+        topics:
+          read:
+            - "topicA"
+            - "topicB"
+      - principal: "User:Connect2"
+        topics:
+          write:
+            - "topicC"
+            - "topicD"
+    topics:
+      - name: "foo"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+      - dataType: "avro"
+        name: "bar"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+  - name: "bar"
+    zookeepers: []
+    consumers: []
+    producers: []
+    streams: []
+    connectors: []
+    topics:
+      - dataType: "avro"
+        name: "bar"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+platform:
+  schema_registry:
+    instances:
+      - principal: "User:SchemaRegistry01"
+      - principal: "User:SchemaRegistry02"
+  control_center:
+    instances:
+      - principal: "User:ControlCenter"
+        appId: "controlcenter"

--- a/src/test/resources/dir_with_subdir/schemas/bar-key.avsc
+++ b/src/test/resources/dir_with_subdir/schemas/bar-key.avsc
@@ -1,0 +1,1 @@
+{"type": "string"}

--- a/src/test/resources/dir_with_subdir/schemas/bar-value.avsc
+++ b/src/test/resources/dir_with_subdir/schemas/bar-value.avsc
@@ -1,0 +1,1 @@
+{"type": "string"}

